IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_MechanicalFastenerOverheadContactSystemRigidSupport/DocPropertySet.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_MechanicalFastenerOverheadContactSystemRigidSupport" UniqueId="37ce8447-2a11-e7c4-7e22-82827dd949f7" ApplicableType="IfcMechanicalFastener" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
Applicable type is general IfcMechanicalFastener? No PDT?

Reply:
Right! de4bcf9de7f5e2e1d9e24e66e3b338da3b5506eb